.gitignore

```diff
@@ -6,3 +6,5 @@ __pycache__
 /dist/
 /resources/build/.vagrant/
 /.pybuild/
+/Testing
```

Comment:
[consider] Not sure about this one. I assume the `Testing` directory is the one created by `ctest`?  The `README` instructions `cd build` before running `ctest`, so I don't think we need to ignore this independent of `build`?

For reference/posterity, there is also `.git/info/exclude` for ignoring local-specific things.